content/releases/3-0-3.md

```diff
@@ -0,0 +1,127 @@
+
+**Change**
+ -  Change WorkMaster parameters input to textbox to support multi-line values better
+ -  Change BPMN Process instances modal default search and width
+ - Refactor Material Class Properties table
+ - Refactor Material Definition table
+ - Refactor Operational Location Class properties table 
+ - Refactor Operational Location properties table 
+ - Refactor Operations Event Class table
+ - Refactor Operations Event Definition to use infinite scroll
+ - Refactor Person Properties table
+ - Refactor Personnel Class table
+ - Refactor Physical Asset Class table
+ - Refactor Physical Asset Classes to use infinite scroll
+ - Refactor Physical Asset table
+ - Refactor Physical assets to use infinite scroll
+ - Refactor Work Masters to use infinite scroll
+ 
+**Fix**
+ -  Fix Person description allowing 'e' character
+ -  Fix Change material definition Version State prompt text
+ -  Fix Create or Select a material definition
+ -  Fix Personnel Class version containers text
+ -  Fix Personnel Class save as hierarchy scope not saving
+ -  Fix typos in person page referring to Physical Asset
+ -  Fix Material Class search
+ -  Fix Operational Location Class hierarchy scope edit
+ -  Fix search on large list of Operational Location Class property
+ -  Fix enable/disable equipment property
+ -  Fix equipment property edit
+ -  Fix equipment property edit query
+ -  Fix Material Class `isAssembledFrom` relationship edits
+ -  Fix BPMN view sidebar title
+ -  Fix equipment nested tree items not showing active version
+ -  Fix user details and support typo
+ -  Fix large number of Person failing to display
+ -  Fix large number of Personnel Classes failing to display
+ -  Fix large number of Operational Location Classes failing to display
+ -  Fix Variable page logic to Create and Set as active
+ -  Fix Operations Definition Segment WorkType selection
+ -  Fix invalid characters in Operations Definition Segment Component
+ -  Fix portal hang when not configured
+ -  Fix Equipment properties text
+ -  Fix numerical error in input fields on Person version for 'Name' and 'Details'
+ -  Fix Operational Location change not refreshing screen after edit
+ -  Fix validation of canCreateNewVersion in Material Class page
+ -  Fix typos in Operational Location page referring to Operational Location Class
+ -  Fix audit trail pdf export header text overlap
+ - Fix cache issue when enabling and disabling Operations Event Definition
+ - Fix previous tag selection script in CI/CD
+ - Fix typo in Create Operational Location
+### BPMN engine
+ 
+No changes for this release
+ 
+### Schema
+
+**Add**
+- Add regex search to Event.messageText
+- Add JobResponseData.valueLong for large values without search
+
+### BAAS
+
+**Fix**
+ - Fix large strings breaking badger indexes by constraining string length to 64000 when indexed and not by hash
+
+### Core
+
+No changes for this release.
```

Comment:
During final testing a we discovered an apollo warning that was resolved. This resulted in a changelog item for libre-core:

```
### Fix
 - Fix supergraph compose warning on JobState enum descriptions
 ```